src/neural/cuda/network_cudnn.cc

```diff
@@ -660,11 +660,11 @@ class CudnnNetwork : public Network {
                         int filterSize) {
     const float epsilon = 1e-5f;
 
-    // Compute reciprocal of std-dev from the variances (so that it can be
-    // just multiplied).
+    // Variance to gamma.
     std::vector<float>& stddev = block.bn_stddivs;
-    for (auto&& w : stddev) {
-      w = 1.0f / std::sqrt(w + epsilon);
+    for (auto i = 0; i < block.bn_stddivs.size(); i++) {
+      block.bn_gammas[i] *= 1.0f / std::sqrt(block.bn_stddivs[i] + epsilon);
+      block.bn_stddivs[i] = 1.0f;
```

Comment:
Is this function still used at all?  I don't think it is - and thus we can just delete it.
Especially since I am very confused about whether the foldBNLayer code path is correct in the presence of gamma folding above.